onboarding/source/amb_light_service.c

```diff
@@ -1,39 +1,125 @@
 #include "amb_light_service.h"
 #include "serial_io.h"
+#include <stdio.h>
 
 #include <adc.h>
 #include <sci.h>
+#include <FreeRTOS.h>
+#include <os_task.h>
+
+#include <os_queue.h>
+#include <os_projdefs.h>
+#include <string.h>
+
+#define Light_Sensor 	6U
 
 /* USER CODE BEGIN */
 // Include any additional headers and global variables here
 
+static TaskHandle_t lightServiceHandle = NULL;
+static QueueHandle_t queue = NULL;
 /* USER CODE END */
 
 /**
  * @brief Task that waits for MEASURE_LIGHT event and then prints the ambient light value to the serial port.
  * @param pvParameters Task parameters
  */
+
+void adcStartConversion_selChn(adcBASE_t *adc, unsigned channel, unsigned fifo_size, unsigned group);
+void adcGetSingleData(adcBASE_t *adc, unsigned group, adcData_t *data);
+uint32 Get_Light_Sensor_data(void);
```

Comment:
Use `uint32_t`. It's the exact same as `uint32`, but it's good to stay consistent.